File: tests/fixtures/resend/resend-marketing-missing-unsubscribe-broken/01-emails-send-no-unsub.ts
import { Resend } from 'resend';

const resend = new Resend(process.env.RESEND_API_KEY);

// Marketing-tagged email with no unsubscribe header and no placeholder in HTML.
export async function sendPromo(email: string) {
  return resend.emails.send({
    from: 'Acme <news@acme.com>',
    to: [email],
    subject: 'Big Summer Sale',
    html: '<h1>50% off everything</h1><p>You opted in.</p>',
    tags: [{ name: 'category', value: 'marketing' }],
  });
}

File: tests/fixtures/resend/resend-marketing-missing-unsubscribe-broken/01-emails-send-no-unsub.py
import os
import resend

resend.api_key = os.environ["RESEND_API_KEY"]

# Marketing-tagged email with no unsubscribe header and no placeholder in HTML.
resend.Emails.send(
    {
        "from": "Acme <news@acme.com>",
        "to": ["user@example.com"],
        "subject": "Big Summer Sale",
        "html": "<h1>50% off everything</h1><p>You opted in.</p>",
        "tags": [{"name": "category", "value": "marketing"}],
    }
)
